sktime/forecasting/compose/tests/test_category_compositor.py

```diff
@@ -0,0 +1,108 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
+"""Tests for the CategoryCompositor"""
+
+import pytest
+
+from sktime.forecasting.compose import CategoryCompositor
+from sktime.forecasting.naive import NaiveForecaster
+from sktime.forecasting.sarimax import SARIMAX
+from sktime.forecasting.trend import PolynomialTrendForecaster
+from sktime.transformations.series.tests.test_adi_cv import (
+    _generate_erratic_series,
+    _generate_smooth_series,
+)
+
+
+@pytest.mark.parametrize(
+    "forecasters, series_generator, chosen_forecaster",
+    [
+        (
+            {
+                "smooth": NaiveForecaster(),
+                "intermittent": SARIMAX(),
```

Comment:
this requires a soft dependency, how about you use `Croston` instead, which is better for intermittent demand anyway?